plugins/plugin-site/src/components/PluginDevelopers.jsx

```diff
@@ -6,7 +6,7 @@ function PluginDevelopers({developers}) {
         <>
             {developers.map((developer) => (
                 <div className="maintainer" key={developer.id}>
-                    {developer.name || developer.id}
+                    <a href={`/ui/search?query=${developer.id}`}>{developer.name || developer.id}</a>
```

Comment:
should be link component. It does some magic to prevent the entire page reloading, just doing react stuff.

Reply:
I can find an example after work, but i think its gatsby-link

Reply:
thanks, fixed